botuser.php

```diff
@@ -221,8 +222,14 @@
                                 $DB = createDBObject();
                                 checkUserInDB($DB, $event["user"]);
                                 $userinfo = getUserInfo($DB,$event["user"]);
+								$userID = getUserInfo($id);
+								$mentor = getMentors($users["users"]);
```

Comment:
What is the $users array? Where does it come from, and what are you expecting it to contain?

Reply:
`getMentors()` is a function. It takes one parameter. The parameter you're giving it is a value in some array called `$users`. This array doesn't exist, so you're passing a value that doesn't exist from an array that doesn't exist to the function